README.md

```diff
@@ -51,17 +51,23 @@ In its current release these are the benchmarks that are executed:
   - `mongodb`: **Deprecated**. [GNU AGPL v3.0](http://www.mongodb.org/about/licensing/)
   - `mongodb_ycsb`: [GNU AGPL v3.0](http://www.mongodb.org/about/licensing/)
   - `netperf`: [HP license](http://www.calculate-linux.org/packages/licenses/netperf)
-  - `oldisim`: Apache v2.
-  - `object_storage_service`: Apache v2.
+  - [`oldisim`](https://github.com/GoogleCloudPlatform/oldisim):
+    [Apache v2](https://github.com/GoogleCloudPlatform/oldisim/blob/master/LICENSE.txt)
+  - [`object_storage_service`](https://github.com/intel-cloud/cosbench):
+    [Apache v2](https://github.com/intel-cloud/cosbench/blob/master/LICENSE)
```

Comment:
@cmccoy, @voellm: just to clarify, do you mean that:

1. I am linking `object_storage_service` to the wrong project, so I should remove or modify the **link**, or 

1. I am linking `object_storage_service` to the right project but you're not using it at all, so I should remove the entire **bullet point**

I am guessing you mean (2) but just wanted to confirm before removing it.